src/schemas/baks/begrunnelse/ks-sak/begrunnelse.tsx

```diff
@@ -0,0 +1,145 @@
+import { apiNavnValideringerBegrunnelse } from '../ba-sak/valideringer';
+import {
+  begrunnelseEØSFlettefelt,
+  begrunnelseFlettefelt,
+  begrunnelseValgfelt,
+} from './begrunnelseFlettefelt';
+
+const editor = (maalform, tittel) => ({
+  name: maalform,
+  title: tittel,
+  type: SanityTyper.ARRAY,
+  of: [
+    {
+      name: DokumentNavn.BLOCK,
+      type: SanityTyper.BLOCK,
+      of: [begrunnelseFlettefelt, begrunnelseEØSFlettefelt, begrunnelseValgfelt],
+    },
+  ],
+});
+
+const begrunnelse = {
+  title: 'Begrunnelse',
+  name: BegrunnelseDokumentNavn.KS_BEGRUNNELSE,
+  type: SanityTyper.DOCUMENT,
+  preview: {
+    select: {
+      title: DokumentNavn.VISNINGSNAVN,
+    },
+  },
+  validation: validerBegrunnelse(),
+  fields: [
+    {
+      title: 'Visningsnavn',
+      type: SanityTyper.STRING,
+      name: DokumentNavn.VISNINGSNAVN,
+      validation: rule => [rule.required().error('Dokumentet må ha et navn')],
+    },
+    {
+      title: 'Behandlingstema',
+      type: SanityTyper.STRING,
+      name: BegrunnelseDokumentNavn.BEHANDLINGSTEMA,
+      options: {
+        list: Object.values(Behandlingstema).map(
+          behandlingstema => behandlingstemaValg[behandlingstema],
+        ),
+      },
+      validation: rule => rule.required().error('Behandlingstema ikke valgt'),
+      initialValue: Behandlingstema.NASJONAL,
+    },
+    {
+      title: 'Begrunnelsetype',
+      type: SanityTyper.STRING,
+      name: BegrunnelseDokumentNavn.BEGRUNNELSE_TYPE,
+      options: {
+        list: Object.values(Begrunnelsestype).map(
+          begrunnelsestype => begrunnelsestyperTilMenynavn[begrunnelsestype],
+        ),
+      },
+      validation: rule => rule.required().error('Begrunnelsestype ikke valgt'),
+    },
+    {
+      title: 'Api-navn',
+      type: SanityTyper.STRING,
+      name: DokumentNavn.API_NAVN,
+      description: 'Teknisk navn. Eksempel innvilgetInnhenteOpplysninger',
+      validation: rule =>
+        apiNavnValideringerBegrunnelse(rule, BegrunnelseDokumentNavn.BA_BEGRUNNELSE),
+    },
+    {
+      title: 'Navn i ba-sak',
+      type: SanityTyper.STRING,
+      name: DokumentNavn.NAVN_I_SYSTEM,
+      validation: rule => [rule.required().error('Dokumentet må ha et navn i ba-sak')],
+    },
+    {
+      title: 'Hjemler',
+      type: SanityTyper.ARRAY,
+      name: BegrunnelseDokumentNavn.HJEMLER,
+      of: [{ type: SanityTyper.STRING }],
+      options: {
+        layout: 'grid',
+        list: hjemler.map(hjemmel => ({ value: hjemmel, title: `§${hjemmel}` })),
+      },
+    },
+    {
+      title: 'Hjemler fra folketrygdloven',
+      type: SanityTyper.ARRAY,
+      name: BegrunnelseDokumentNavn.HJEMLER_FOLKETRYGDLOVEN,
+      of: [{ type: SanityTyper.STRING }],
+      options: {
+        layout: 'radio',
+        list: hjemlerFolketrygdloven.map(hjemmel => ({ value: hjemmel, title: `§${hjemmel}` })),
+      },
+    },
+    ...eøsHjemler,
```

Comment:
Samme hjemler for KS som for BA?

Reply:
Hehe, nei. Snakket med Eivind og vi ble enige om å ha de som ba enn så lenge siden løypa om hva vi trenger ikke er gått opp helt enda